Parsing error on `logo.html` in Slinkity docs

**Describe the bug**
A strange parsing error occurs on our doc's `logo.html`.

```bash
11ty] Copied 33 files / Wrote 8 files in 0.37 seconds (v1.0.0-canary.41)
[11ty] Watching…
[11ty] Unhandled rejection in promise: (more in DEBUG output)
[11ty] > Unable to parse {"file":"/index.html","line":137,"column":7}
135|          <stop offset="1" stop-color="#FAFF02" ></stop>
136|        
137|        <linearGradient id="paint8_linear" x1="265.37" y1="-9.04662e-06" x2="148.808" y2="126.632"
   |        ^
138|          gradientUnits="userSpaceOnUse">
139|          <stop stop-color="#FF00D6" ></stop>

`Error` was thrown:
[11ty]     Error: Unable to parse {"file":"/index.html","line":137,"column":7}
    135|          <stop offset="1" stop-color="#FAFF02" ></stop>
    136|        
    137|        <linearGradient id="paint8_linear" x1="265.37" y1="-9.04662e-06" x2="148.808" y2="126.632"
       |        ^
    138|          gradientUnits="userSpaceOnUse">
    139|          <stop stop-color="#FF00D6" ></stop>
        at traverseHtml (/Users/benholmes/Repositories/slinkity/docs/node_modules/vite/dist/node/chunks/dep-85dbaaa7.js:21240:15)
        at processTicksAndRejections (internal/process/task_queues.js:95:5)
        at async devHtmlHook (/Users/benholmes/Repositories/slinkity/docs/node_modules/vite/dist/node/chunks/dep-85dbaaa7.js:56997:5)
        at async applyHtmlTransforms (/Users/benholmes/Repositories/slinkity/docs/node_modules/vite/dist/node/chunks/dep-85dbaaa7.js:21550:21)
        at async viteTransformMiddleware (/Users/benholmes/Repositories/slinkity/docs/node_modules/slinkity/lib/plugin/index.js:135:23)
^Cnpm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! slinkity-docs@1.0.0 start: `slinkity --serve --incremental`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the slinkity-docs@1.0.0 start script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/benholmes/.npm/_logs/2021-10-31T22_05_32_698Z-debug.log
```

**To Reproduce**
Steps to reproduce the behavior:
1. Go to the `docs` directory of this project
2. Run `npm start`
3. Notice a parsing error occurs on `index.html` (and any other file for that matter, if you delete `index.md`)

**Expected behavior**
Parsing should continue to work in-line with regular 11ty. Note that the project compiles when using plain 11ty!

**(please complete the following information):**
 - OS: OSX 11.5.1 (20G80)
 - Browser N/A
 - Version 0.2.0


**Additional context**
Looks like [Node HTML parser](https://www.npmjs.com/package/node-html-parser) just launched a new major version. Doing some local testing, it seems SVGs are parsed incorrectly on the latest v4. This behavior should be fixed when upgrading to v5.

--- a/package.json
+++ b/package.json
@@ -1,7 +1,7 @@
 {
   "name": "slinkity",
   "description": "To 11ty and beyond! The all-in-one tool for templates where you want them, component frameworks where you need them",
-  "version": "0.2.0",
+  "version": "0.2.1",
   "license": "MIT",
   "main": "./lib/plugin/index.js",
   "homepage": "https://slinkity.dev",

--- a/package.json
+++ b/package.json
@@ -56,7 +56,7 @@
     "glob": "^7.1.7",
     "html-react-parser": "^1.2.7",
     "javascript-stringify": "^2.1.0",
-    "node-html-parser": "^4.1.3",
+    "node-html-parser": "^5.1.0",
     "require-from-string": "^2.0.2",
     "sass": "^1.36.0",
     "vite": "^2.4.4"

